events/voiceStateUpdate.js

```diff
@@ -95,9 +95,10 @@ module.exports = {
 				// Quaver was playing something - set pauseTimeout
 				await player.pause();
 				logger.info({ message: `[G ${player.guildId}] Setting pause timeout`, label: 'Quaver' });
-				if (player.pauseTimeout) {
-					clearTimeout(player.pauseTimeout);
-				}
+				// Ensure that the bot does not set a new pauseTimeout if pauseTimeout already exists
+				if (player.pauseTimeout) return;
+				// When setting a pauseTimeout, clear pauseTimeout at any cost as failsafe
+				clearTimeout(player.pauseTimeout);
```

Comment:
`player.pauseTimeout` is always `undefined` here. Because everytime the user resumes its session, pauseTimeout gets cleared. This is a redundant statement and will for now stay here as a failsafe.